stix_shifter_modules/alertflex/entry_point.py

```diff
@@ -0,0 +1,54 @@
+from stix_shifter_utils.utils.base_entry_point import BaseEntryPoint
+from stix_shifter_utils.modules.base.stix_transmission.base_sync_connector import BaseSyncConnector
+from .stix_transmission.ping_connector import PingConnector
+from .stix_transmission.delete_connector import DeleteConnector
+from .stix_transmission.results_connector import ResultsConnector
+from .stix_transmission.api_client import APIClient
+from .stix_translation.query_translator import QueryTranslator
+from stix_shifter_utils.stix_translation.src.json_to_stix.json_to_stix import JSONToStix
+import os
+
+
+class EntryPoint(BaseEntryPoint):
```

Comment:
`EntryPointBase`

Reply:
I have checked it, 

In early releases until 3.0.9 (including this release)
you used next code (see adapter synchronous_dummy, module entry_point.py) :

from stix_shifter_utils.utils.entry_point_base import EntryPointBase
class EntryPoint(EntryPointBase)

Since release 3.1.0 you use:

from stix_shifter_utils.utils.base_entry_point import BaseEntryPoint
class EntryPoint(BaseEntryPoint)

it was a reason why alertflex adapter has shown the message "the module not found error" on the run,
because it was written for early releases. Alertflex adapter with commit 52335a1 should work
with your new release 3.1.0 (stix-shifter-develop branch) as far I understand your stix-shifter-master branch
was also recently (3 days ago) updated to this release.

Reply:
yes we had some back and forth refactoring. and we finally merged develop into master. and your develop branch looks good so far.